Implementation of nf-gpt into crisprseq pipeline. (#193)
nf-core/crisprseq: docs/usage/screening.md

> [!WARNING] Please be advised that the Hitselection algorithm is time intensive and will make the pipeline run longer

### Running nf-gpt

Data from screening analysis can be interpreted using [nf-gpt](https://github.com/nextflow-io/nf-gpt). Currently MAGeCK MLE, MAGeCK RRA, BAGEL2 and drugZ are supported.

> To run this you can use `--gpt_interpretation` followed by the tools, whose data you want to interprete. For example `--gpt_interpretation mle,rra,bagel2,drugz` runs the interpretation on all 4 tools. You can also run less, or single tools as well. Since the plugin works through the OpenAI api, you will have to make sure that there is a valid api key in your local environment. For this use `export OPENAI_API_KEY=your-api-key` in your project environment prior to running the pipeline.

Reviewer: ```suggestion

To run this you can use `--gpt_interpretation` followed by the name of the tools which you want to interprete. 
For example `--gpt_interpretation mle,rra,bagel2,drugz` runs the interpretation on all 4 tools. 

Since the plugin works through the OpenAI API, you will have provide a valid API key in your local environment. 
For this, use `export OPENAI_API_KEY=your-api-key` in your project environment prior to running the pipeline.
```

> [!WARNING]

> > In order for `nf-gpt` to run successfully, you will also have to run the corresponding tool. Simply specifying the tool with the `gpt_interpretation` parameter will **not** run the tool in the pipeline. Therefore, make sure to also run the tool using `--mle`, `--rra`, `bagel2` and `drugz`.

Reviewer: ```suggestion
> In order for `nf-gpt` to run successfully, you will also have to run the corresponding tool. Simply specifying the tool with the `gpt_interpretation` parameter will **not** run the tool in the pipeline. Therefore, make sure to also run the tool using `--mle`, `--rra`, `--bagel2` and/or `--drugz`.
```

> [!WARNING]

> > Be adviced that gpt produced text data has no guarantee to be correct. There have been various instances were LLM outputs have been proven to be the result of hallucination. Running `nf-gpt` does **not** guarantee scientifically correct results and is for now simply a experimental inclusion.

Reviewer: ```suggestion
> Be adviced that gpt produced text data has no guarantee to be correct. There have been various instances where LLM outputs have been proven to be the result of hallucination. Running `nf-gpt` does **not** guarantee scientifically correct results and is an experimental tool.
```
